hw/hw5/main.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import FastAPI, HTTPException
+from typing import List
+from pydantic import BaseModel
+
+
+class Task(BaseModel):
+    id: int
+    header: str
+    description: str
+    done: bool
+
+
+app = FastAPI()
+db = list()
+
+
+@app.get("/tasks/", response_model=List[Task])
+async def get_tasks():
+    return db
+
+
+@app.get("/tasks/{task_id}", response_model=Task)
+async def get_task(task_id: int):
+    for i in range(len(db)):
+        if db[i].id == task_id:
+            return db[i]
```

Comment:
Nne можно просто пройтись по списку
```suggestion
    for task in db:
        if task.id == task_id:
            return task
```